### PR TITLE
Friday tweaks

### PR DESCRIPTION
**Pull request by SpamapS**
Hacking on a Friday. This gives us a test suite. Maybe we can have travis CI run it?

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,32 @@
 venv
+AUTHORS
```
Comment (mlangbehn) [inline .gitignore:2]: Why are we ignoring Authors?

File: .gitignore [hunk 2]
```diff
@@ -1 +1,32 @@
...
+build/*
+build-stamp
+ChangeLog
```
Comment (mlangbehn) [inline .gitignore:5]: Why are we ignoring ChangeLog?

**Comment by nibalizer** [comment]
This was created before we dropped environment awareness. So this needs to be updated to not know about environments or we need to add environments back. My sense is that the git based finglonger workflow doesn't work with the basic environment stuff that I added, so i'd prefer this to be updated to not use enviroments.

**Comment by nibalizer** [comment]
I should say, I love this patch. :heart:

File: .gitignore (hunk 1 above)
Comment (SpamapS) [reply .gitignore:2]: Because it is available in git logs.

File: .gitignore (hunk 2 above)
Comment (SpamapS) [reply .gitignore:5]: Same as AUTHORS

**Comment by SpamapS** [comment]
Not sure I understand what you mean by environments. I want to expand on this and get even more test capability in, but would like to see this land first.

**Comment by eggshell** [comment]
@SpamapS #13. The only test that explicitly references environments is `get_test_config()`

**Comment by nibalizer** [comment]
Reuploaded as #19

**Pull request closed without merging**